Source/ACE.Database/ShardDatabase.cs

```diff
@@ -115,7 +116,7 @@ public List<(uint start, uint end)> GetSequenceGaps(uint min, uint limitAvailabl
 
                 while (reader.Read())
                 {
-                    var gap_starts_at               = reader.GetFieldValue<double>(0);
+                    var gap_starts_at               = reader.GetFieldValue<long>(0);
                     var gap_ends_at_not_inclusive   = reader.GetFieldValue<decimal>(1);
```

Comment:
Should the second field be long as well? I think so.

Reply:
according to the ready the second field is still decimal, and the third was unchanged as well